src/web/app/pages-instructor/instructor-courses-page/course-permanent-deletion-confirm-modal/course-permanent-deletion-confirm-modal.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="modal-header bg-danger text-white">
+  <h5 class="modal-title"><i class="fas fa-times-circle"></i> Confirm deleting a course</h5>
+  <button type="button" class="close" (click)="activeModal.dismiss()">
+    <i class="fas fa-times"></i>
+  </button>
+</div>
+<div class="modal-body">
+  <strong>Are you sure you want to permanently delete {{courseId}}?</strong>
+  <br>
+  This operation will delete all students and sessions in these courses.
+  All instructors of these courses will not be able to access them hereafter as well.
```

Comment:
For this section, all instances of "these courses" should just be "this course".